include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -224,6 +224,10 @@ class format_parse : public format_base
     template <typename id_type>
     std::vector<std::string>::iterator find_option_id(std::vector<std::string>::iterator const begin_it, id_type const & id)
     {
+        if constexpr (std::Same<remove_cvref_t<id_type>, std::string>)
+            if (id.empty())
+                return end_of_options_it;
```

Comment:
what if id_type is not a string? 🤔

Reply:
Then it is a char and never empty but maybe the null character which cannot be supplied via the command line so there is never a '-\0' option and the fine command is fine.

Reply:
> Then it is a char and never empty but maybe the null character which cannot be supplied via the command line 

Sure you can supply it, e.g. by copy-n-paste or by invocation from a script... That's why we need fuzzy testing of the argument_parser!